### ISSUE TITLE
Error on install

### ISSUE THREAD

**Issue by covard** [docmunch/homebrew-pdftk#3]
Getting an error installing this. I followed the paths and I don't have the ``` x86_64-apple-darwin14.1.0 ``` directory in ``` /usr/local/Cellar/gcc48/4.8.4/lib/gcc/ ```

OSX 10.10.2 (it's a brand new machine)

Error:

``` shell
==> make -f Makefile.OSX-10.6
/usr/local/bin/g++-4.8 -Wall -Wextra -Weffc++ -O2 attachments.o report.o passwords.o pdftk.o /private/tmp/pdftk-AupCJ2/pdftk/../java/java_lib.o  /usr/local/Cellar/gcc48/4.8.4/lib/gcc/x86_64-apple-darwin14.1.0/4.8.4/libgcj.dylib /usr/local/Cellar/gcc48/4.8.4/lib/gcc/x86_64-apple-darwin14.1.0/4.8.4/libstdc++.dylib /usr/local/Cellar/gcc48/4.8.4/lib/gcc/x86_64-apple-darwin14.1.0/4.8.4/libgcc_s.1.dylib -liconv -lz -o pdftk
g++-4.8: error: /usr/local/Cellar/gcc48/4.8.4/lib/gcc/x86_64-apple-darwin14.1.0/4.8.4/libgcj.dylib: No such file or directory
g++-4.8: error: /usr/local/Cellar/gcc48/4.8.4/lib/gcc/x86_64-apple-darwin14.1.0/4.8.4/libstdc++.dylib: No such file or directory
g++-4.8: error: /usr/local/Cellar/gcc48/4.8.4/lib/gcc/x86_64-apple-darwin14.1.0/4.8.4/libgcc_s.1.dylib: No such file or directory
make: *** [pdftk] Error 1
```

**Comment by spl** [comment]
@covard Thanks. Is this similar to the hardcoded fix that #2 provides? I looked at that briefly a while back. I'll try to look into it again soon.

**Comment by covard** [comment]
Yes it does. The hardcoded path in https://github.com/docmunch/homebrew-pdftk/pull/2 looks like it would fix it. Thanks.

**Comment by spl** [comment]
@covard @pborenstein Can you please try to install the `pdftk.tb` below and tell me if it works? If it does, I'll release it. Thanks!

The changes are:

* Use `gcc` as a dependency instead of `gcc48`
* Remove `:build` from the dependencies, since (I think) `pdftk` needs the Java libs even after building
* Fix the paths used in the `Makefile`

```
brew uninstall pdftk
brew install https://raw.githubusercontent.com/docmunch/homebrew-pdftk/use-gcc-fix-dirs/pdftk.rb
```

**Comment by covard** [comment]
This worked. Thanks for the update.

**Comment by pborenstein** [comment]
Looks OK. Here's my install log:

https://gist.github.com/pborenstein/4ff09effbbd591604fe7

**Comment by spl** [comment]
Closed by #4, which adds a few more goodies. Thanks for your help. Sorry for the delay in responding.
